model/version.go

```diff
@@ -138,7 +139,7 @@ func (self *Version) SetActivated() error {
 }
 
 func (self *Version) SetNotActivated() error {
-	if utility.FromBoolPtr(self.Activated) {
+	if self.Activated != nil {
```

Comment:
It's possible we'll want to use this function in the future to set something from activated to not activated (not sure why but it does seem like something that should work), so I think we only really want to return early if self.Activated is set explicitly to False already (you can use !utility.FromBoolTPtr to this, because the nil case defaults to true)